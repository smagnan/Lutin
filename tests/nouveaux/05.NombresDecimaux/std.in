0.5
0.05

