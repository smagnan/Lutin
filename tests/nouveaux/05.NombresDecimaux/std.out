EXECUTING PROGRAM :
0.0500000000000000
0.5500000000000000
0.4500000000000000
0.0250000000000000
10.0000000000000000
